realm/realm-library/src/androidTest/java/io/realm/RealmConfigurationTests.java

```diff
@@ -108,6 +102,30 @@ public void setDefaultConfiguration_nullThrows() throws NoSuchFieldException, Il
     }
 
     @Test
+    public void getDefaultConfiguration_returnsTheSameObjectThatSetDefaultConfigurationSet() {
+        final RealmConfiguration defaultConfiguration = Realm.getDefaultConfiguration();
```

Comment:
Doing this implies that a default configuration is usable across multiple unit tests, which definitely shouldn't be the case. Each test should be isolated. So even though it is correct, it just adds noise IMO?